cypress/readonly/integration/authentication/login.cy.ts

```diff
@@ -7,21 +8,14 @@ describe('Given a user with a DAWSON account', () => {
 
   describe('When they login in with the correct email and password', () => {
     it('Then they should be taken to their dashboard', () => {
-      cy.visit('/login');
-      cy.get('[data-testid="email-input"]').type(
-        'testAdmissionsClerk@example.com',
-      );
-      cy.get('[data-testid="password-input"]').type(
-        getCypressEnv().defaultAccountPass,
-      );
-      cy.get('[data-testid="login-button"]').click();
-      cy.get('[data-testid="account-menu-button"]');
-      cy.get('[data-testid="error-alert"]').should('not.exist');
+      loginAsAdmissionsClerk();
```

Comment:
We can't change the user this uses because `admissionsclerk1@example.com` doesn't exist on prod.

```suggestion
      loginAsAdmissionsClerk('testAdmissionsClerk@example.com');
```